### DISCUSSION TITLE
Domain Bypass

### DISCUSSION THREAD

**Discussion by sah66214** [Ysurac/openmptcprouter#3104] [General]
Hi
im noob but thanks to yuric that i can run this . very useful .
i want to bypass some domain that go through normal internet , not vpn&vps . i try omr-bypass but it does not work .
apperciate for any help

**Comment by Ysurac** [comment]
To bypass a domain, OpenMPTCProuter IP must be used as DNS on client.
If it's already the case, wait for DNS cache to timeout, or reboot your client.

**Reply by Melania95** [reply]
Hi Ysurac, I did what you indicated but I noticed that it no longer uses the IP of my VPS but that of the selected ISP (If I do a speedtest for example it doesn't give me the IP of my Vps)... is this normal? (Thank you for everything you have done and are doing for all of us!)
Edit: I use 0.59.1 kernel 5.4
